app/src/gui/components/notebook/overview_map.tsx

```diff
@@ -139,44 +140,94 @@ export const OverviewMap = (props: OverviewMapProps) => {
   const mapRef = useRef<Map | undefined>();
   mapRef.current = map;
 
-  const map_center = [30, -10];
+  const {data: map_center} = useQuery<Array<number>>({
```

Comment:
nitpick: react-query course advises type annotating, if at all, as close to the data fetch as possible. I believe the generics on useQuery can muddle things up - so can just remove the Array<number> here, or if needed annotate line 147.

Reply:
I know I've done this too

Reply:
Moved the types to the queryFn